galoy.yaml

```diff
@@ -207,3 +217,7 @@ skipFeeProbeConfig:
   chanId: []
 smsAuthUnsupportedCountries: []
 whatsAppAuthUnsupportedCountries: []
+appcheckConfig:
```

Comment:
if this is related to firebase and depending on the service that you use is better to rename it. maybe firebaseAppcheckConfig?

Reply:
the product is called App Check but you can write custom providers (i.e for hauwei )